packages/sdk/src/Products.ts

```diff
@@ -7,67 +9,73 @@ export class Products {
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#create-product
 	 */
-	public create(product: $FixMe) {
-		return this.client.post('/products', product)
+	public create(product: T.ProductsCreate) {
+		return this.client.post<T.ProductsCreateResponse>('/products', product)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#get-product
 	 */
-	public get(productId: $FixMe) {
-		return this.client.get(`/products/${encode(productId)}`)
+	public get(productId: string) {
+		return this.client.get<T.ProductsGetResponse>(`/products/${encode(productId)}`)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#update-product
 	 */
-	public update(product: $FixMe) {
-		return this.client.put(`/products/${encode(product.id || product.source_id)}`, omit(product, ['id']))
+	public update(product: T.ProductsUpdate) {
+		return this.client.put<T.ProductsUpdateResponse>(
+			`/products/${encode(product.id || product.source_id)}`,
+			omit(product, ['id']),
+		)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#update-products-in-bulk
 	 */
-	public bulkUpdate(products: $FixMe) {
-		return this.client.post('/products/bulk', products)
+	public bulkUpdate(products: T.ProductsBulkUpdate) {
+		return this.client.post<T.ProductsBulkUpdateResponse>('/products/bulk', products)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#delete-product
 	 */
-	public delete(productId: $FixMe, params?: $FixMe) {
+	public delete(productId: string, params?: T.ProductsDeleteParams) {
 		return this.client.delete(`/products/${encode(productId)}`, { qs: params })
```

Comment:
This is an error, as second parameter it should just be `params` and not `{ qs: params}`.